tutorials/astro_dark_matter.ipynb

```diff
@@ -268,6 +275,19 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.7.0"
+  },
```

Comment:
Do you know why this meta data is added? Is it because you use a specific Jupyer version? Do you now whether it's compatible with "standard" jupyter?

Reply:
This meta data accounts for the python kernel used in the notebook. I have several kernels declared for the different conda versions of gammapy installed locally, and a `gammapy-dev` kernel for the development version. When in other desktop we try to execute a notebook with an already declared kernel in the metas, we will have a kernel not found error if this one is not found. 

In the case we still want to provide support for Python 2.7, I think we should remove the whole `kernelspec` keyword in the meta dictionary. For the moment I will stick to the Python3 standard kernel, as we do for the others.

Reply:
OK, thanks for the explanation.